src/common/components/Recaptcha/RecaptchaWrapper.tsx

```diff
@@ -50,11 +52,23 @@ export const RecaptchaWrapper = ({
     document.body.appendChild(script)
   }, [])
 
+  // Because The onSuccess and onFailure callbacks are called by a React component, AngularJS doesn't know that an event happened and doesn't know it needs to rerender. We have to use $apply to ensure that AngularJS rerenders after the event handlers return.
+  const onSuccessWrapped = (() => {
+    $rootScope.$apply(()=> {
+      onSuccess.call(componentInstance)
+    })
+  })
+
+  const onFailureWrapped = (() => {
+    $rootScope.$apply(()=> {
+      onFailure.call(componentInstance)
+    })
+  })
```

Comment:
Looks like `standardjs` doesn't run on TSX files.

```suggestion
  const onSuccessWrapped = (() => {
    $rootScope.$apply(() => {
      onSuccess.call(componentInstance)
    })
  })

  const onFailureWrapped = (() => {
    $rootScope.$apply(() => {
      onFailure.call(componentInstance)
    })
  })
```